modules/integration/tests-integration/tests-amqp/src/test/java/org/wso2/mb/integration/tests/amqp/functional/QueueMessageExpireTestCase.java

```diff
@@ -178,16 +185,38 @@ public void performManyQueueExpirySendReceiveTestCase()
         AndesClient publisherClientWithExpiration = new AndesClient(publisherConfigWithExpiration, true);
         publisherClientWithExpiration.startClient();
 
-        AndesClientUtils.waitForMessagesAndShutdown(initialConsumerClient, AndesClientConstants.DEFAULT_RUN_TIME);
-        AndesClientUtils.waitForMessagesAndShutdown(secondaryConsumerClient, AndesClientConstants.DEFAULT_RUN_TIME);
+        long consumer1MsgCount = initialConsumerClient.getReceivedMessageCount();
+        long consumer2MsgCount = secondaryConsumerClient.getReceivedMessageCount();
+        long timeout = 600000L;
```

Comment:
this is the timeout. in normal condition, it shouldn't hit the timeout. if this timeout hits(there's a error log added) we can confirm there are messages getting accumulated in Jenkins build and investigate on that path.